manager/src/main/java/org/apache/hertzbeat/manager/service/impl/PluginServiceImpl.java

```diff
@@ -335,18 +428,30 @@ private List<URL> loadLibInPlugin(String pluginJarPath) {
                         out.flush();
                     }
                 }
+                if ((entry.getName().contains("define")) && (entry.getName().endsWith(".yml") || entry.getName().endsWith(".yaml"))) {
+                    try (InputStream ymlInputStream = jarFile.getInputStream(entry)) {
+                        List<ParamDefine> params = yaml.loadAs(ymlInputStream, List.class);
+                        PARAMS_Define_MAP.putIfAbsent(pluginMetadataId, params);
+                    }
+                }
```

Comment:
It is necessary to add the plugin parameter configuration file content check function to upload Plugin. In my previous test, if define.yml has an error, if the user installs a plugin containing an incorrect configuration file, the manager will not be able to restart.